spec/javascripts/govuk-component/govspeak-enhance-youtube-video-links-spec.js

```diff
@@ -0,0 +1,67 @@
+describe('Links to YouTube in govspeak content', function () {
+  "use strict";
+
+  var enhanceYoutubeVideoLinksFixture, $enhanceYoutubeVideoLinksHTML, enhanceYoutubeVideoLinks;
+
+  var turnOffEnhanceYoutubeVideoLinksFixture, $turnOffEnhanceYoutubeVideoLinksHTML;
+
+  describe('- A YouTube link to be converted to an embedded player', function () {
+
+    beforeEach(function() {
+
+      enhanceYoutubeVideoLinksFixture = '<div class="govuk-govspeak">'+
+                      '<a href="https://youtu.be/OzjogCFO4Zo">'+
+                        'This link to YouTube'+
+                      '</a>'+
+                    '</div>';
+
+      $enhanceYoutubeVideoLinksHTML = $(enhanceYoutubeVideoLinksFixture);
+      $('body').append($enhanceYoutubeVideoLinksHTML);
+      GOVUK.enhanceYoutubeVideoLinks($(".govuk-govspeak"));
+    });
+
+    afterEach(function() {
+      $('.govuk-govspeak').remove();
```

Comment:
Not necessary to change this, but you could instead call `$enhanceYoutubeVideoLinksHTML.remove()`